app/scripts/modules/core/src/reactShims/react.injector.ts

```diff
@@ -48,6 +46,8 @@ import { VariableInputService } from '../pipeline/config/templates/inputs/variab
 import { VariableValidatorService } from '../pipeline/config/templates/validators/variableValidator.service';
 import { ViewStateCacheService } from '../cache/viewStateCache.service';
 import { WaypointService } from '../utils/waypoints/waypoint.service';
+import { VersionSelectionService } from '../../lib/cloudProvider/versionSelection/versionSelection.service';
```

Comment:
I think this is up one too many and then back down. (i.e. could be `'../cloudProvider/versionSelection/..'`)